### PR TITLE
Sp lo

### PR DESCRIPTION
**Pull request by staceypark**
I tried making some edits, but when it came to the actual models, I don't feel like there's much we can cut without losing the lesson?  I took out some items that I felt read redundant.

### CODE REVIEW

File: facilitation_guide/learning_objectives.rmd [hunk 1]
```diff
@@ -1,21 +1,18 @@
+
 Session 1 - Our Vision  
-1.	Explain how Modeling to Learn seeks to align the health care team in working together to improve quality of care.  
-2.	Articulate a team need that MTL will address.  
-3.	Recognize the interdependence of quality of work life and quality of care.  
-4.	Describe how MTL helps providers see and understand their work within VA data, standards and initiatives.  
+1.	Explain how Modeling to Learn will help align the team together to improve both quality of care amd quality of work life
+2.	Identify a specific team need that MTL will address  
+3.  Make VA data and intiatives transparent in the context of providers' work
```
Comment (branscombj) [inline facilitation_guide/learning_objectives.rmd]: Remember that LOs are not the facilitator's objectives to cover during the session, but should complete the sentence, "After the session, the participant will be able to...". So 1 and 2 can translate to that, but 3 probably needs tweaking.

File: facilitation_guide/learning_objectives.rmd [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
 
 Session 2. Our Data  
-1.	Demonstrate how to access and update the team’s patient data on the SharePoint site.  
-2.	Identify a specific patient in the team data.  
-3.	Describe how the team data are categorized into visits by CPT Codes.  
-4.	Specify the team’s preferred data for Modeling to Learn.  
-5.	Describe the limits of retrospective data review for guiding change.  
+1.	Teach the team how data tool works - filtering data, CPT codes, updating data, accessing data, etc.
```
Comment (branscombj) [inline facilitation_guide/learning_objectives.rmd]: same as comment above

File: facilitation_guide/learning_objectives.rmd [hunk 3]
```diff
@@ -1,21 +1,18 @@
...
+2.	Identify a specific patient in the team data to make the data more personable.
```
Comment (branscombj) [inline facilitation_guide/learning_objectives.rmd]: I think the participant needs to be able to find a particular patient within the data tool - to "zoom in" and "zoom out" so they can appreciate that the aggregate data is made up of actual, individual patients. If this is the intent, then the LO could just be "Identify a specific patient within the data using the team data tool." or something.

**Comment by TomRust** [comment]
It looks like Jane rejected these changes, but didn't close the pull request.

**Pull request closed without merging**